go/ql/test/library-tests/semmle/go/frameworks/GoMicro/client/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	pb "codeql-go-tests/frameworks/GoMicro/proto"
+	"context"
+	"fmt"
+	"log"
+
+	micro "go-micro.dev/v4"
+)
+
+func main() {
+	// service
+	service := micro.NewService()
+	service.Init()
+	// context
+	ctx := context.Background()
+
+	greeterService := pb.NewGreeterService("http://localhost:8000", service.Client())
```

Comment:
Switch to inline expectations test
```suggestion
	greeterService := pb.NewGreeterService("http://localhost:8000", service.Client()) // $ clientRequest="http:\/\/localhost:8000"
```